components/bl-line-chart-component/README.md

```diff
@@ -0,0 +1,40 @@
+# Line Chart
+The Line Chart component allows you to display data in the form of charts. You can also choose a chart type. Also, the [chart.js](https://www.chartjs.org/) library is used to create this component.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property                | Type                                                   | Default Value                                                                                                                                                                                                                                                                                                                                                          | Logic          | Data Binding | UI Setting | Description                                                                                                                                                                                                                                                                         |
+|-------------------------|--------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|--------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Disabled<br/>`disabled` | Checkbox                                               | `false`                                                                                                                                                                                                                                                                                                                                                                | Disabled Logic | YES          | YES        | This handler allows you to disable a component.                                                                                                                                                                                                                                     |
```

Comment:
Need to add basic styling settings

font
background colors
grid color
line width
border
etc.

Please take a look which properties will be easy to implement.